src/main/scala/de/htwg/se/sudoku/aview/Tui.scala

```diff
@@ -51,7 +56,12 @@ class Tui(controller: ControllerInterface) extends Reactor with LazyLogging{
   def printCandidates: Unit = {
     logger.info("Candidates: ")
     for (row <- 0 until size; col <- 0 until size) {
-      if (controller.isShowCandidates(row, col)) println("("+row+","+col+"):"+controller.available(row, col).toList.sorted)
+      if (controller.isShowCandidates(row, col))
+        println(
+          "(" + row + "," + col + "):" + controller
+            .available(row, col)
+            .toList
+            .sorted)
```

Comment:
is this auto formatted? looks kinda strange

Reply:
yep, `scalafmt` with default settings...think to change the IDE's to different rules should be avoided.